kogito-benchmarks-framework/src/main/java/org/kie/kogito/benchmarks/framework/Commands.java

```diff
@@ -150,18 +152,32 @@ public static String getArchivedLogsDir() {
         return getSystemPropertyOrEnvVarValue("archivedLogsDir");
     }
 
-    private static String getSystemPropertyOrEnvVarValue(String name) {
+    public static boolean getUseCustomMavenRepo() {
+        String value = getSystemPropertyOrEnvVarValueSilent("useCustomMavenRepo");
+        return !"false".equalsIgnoreCase(value);
```

Comment:
so if the value is `"bananas"`, then this returns true?

Reply:
Yes, I was thinking about this quite a lot, what should be the default etc. So basically you have to explicitly provide `<useCustomMavenRepo>false</useCustomMavenRepo>` to turn it off. I was also thinking about providing a method which would check if the property is either not specified (null), or set to true/false, but I would say it's quite an overkill. Basically it's the opposite of how `Boolean.parseBoolean` works as it only returns true if the string is true; if it is null, false or `bananas`, it will return false.
I could do it otherwise using `dontUseCustomMavenRepo` or `useDefaultMavenRepo`, but I think this way the meaning of the properties is less clear.